nilearn/surface/surface.py

```diff
@@ -512,6 +514,51 @@ def vol_to_surf(img, surf_mesh,
     return texture.T
 
 
+def _load_surf_files_gifti_gzip(surf_file):
+    """Load surface data Gifti files which are gzipped. This
+    function is used by load_surf_mesh and load_surf_data for
+    extracting gzipped files.
+
+    Part of the code can be removed while bumping nibabel 2.0.2
+    """
+    with gzip.open(surf_file) as f:
+        as_bytes = f.read()
+    if LooseVersion(nibabel.__version__) >= LooseVersion('2.1.0'):
+        parser = gifti.GiftiImage.parser()
+        parser.parse(as_bytes)
+        gifti_img = parser.img
+    else:
+        from nibabel.gifti.parse_gifti_fast import ParserCreate, Outputter
+        parser = ParserCreate()
+        parser.buffer_text = True
+        HANDLER_NAMES = ['StartElementHandler', 'EndElementHandler',
```

Comment:
here maybe I would write all three assignments explicitely since it results in less lines ie
```
parser.StartElementHandler = out.StartElementHandler
parser.EndElementHandler = out.EndElementHandler
parser.CharacterDataHandler = out.CharacterDataHandler
```
instead of `HANDLER_NAMES = ...` and the loop
or at least replace `HANDLER_NAMES` -> `handler_names` since it is a local variable

sorry for not doing this in what I pasted earlier